pvr.mediaportal.tvserver/resources/language/resource.language.en_gb/strings.po

```diff
@@ -66,7 +66,11 @@ msgctxt "#30011"
 msgid "Enable old series recording dialog"
 msgstr ""
 
-#empty strings from id 30012 to 30014
+msgctxt "#30012"
+msgid "Default lifetime"
```

Comment:
Could you make msgid "Default recording lifetime" and clear msgstr (similar to the other items)
Similarly, change the msgid fields in your German translation.
Note that all translations are done on Transifex, so witht the next merge, the German translation will be gone again.